src/pm/hydra/tools/topo/hwloc/topo_hwloc.c

```diff
@@ -424,9 +424,10 @@ HYD_status HYDT_topo_hwloc_init(const char *binding, const char *mapping, const
     else if (!strncmp(membind, "interleave:", strlen("interleave:"))) {
         HYDT_topo_hwloc_info.membind = HWLOC_MEMBIND_INTERLEAVE;
     }
-    else if (!strncmp(membind, "replicate:", strlen("replicate:"))) {
+    /*else if (!strncmp(membind, "replicate:", strlen("replicate:"))) {
         HYDT_topo_hwloc_info.membind = HWLOC_MEMBIND_REPLICATE;
-    }
+  	}
+  	*/
```

Comment:
Please just delete this instead of commenting it out.  That's what version control systems are for.